src/client/components/form-elements/DateInput.jsx

```diff
@@ -58,9 +58,18 @@ class DateInput extends React.Component {
           validateIfDirtyDate(day, month, year, isValidDate);
     }
 
+    // Calculate required.
+    let requiredSpan = undefined;
+    if (this.props.required) {
+      requiredSpan = <span className="hca-required-span"> *</span>;
+    }
+
```

Comment:
Yes and when we write documentation we can use code examples to show people how to use it.